### PR TITLE
php70-uopz: 5.0.1 (new formula)

### PR DESCRIPTION
**Pull request by SMillerDev**
- [*] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-php/pulls) for the same formula update/change?
- [*] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

-----

This formula adds uopz with php7.0 support, the only error is that the description is too long, it's copied from the 56 formula however (and a little shortened)

### CODE REVIEW

File: Formula/php70-uopz.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+require File.expand_path("../../Abstract/abstract-php-extension", __FILE__)
+
+class Php70Uopz < AbstractPhp70Extension
+  init
+  desc "Exposes Zend Engine functionality normally used at compilation/execution time."
```
Comment (javian) [inline Formula/php70-uopz.rb]: Please fix this
```
Error: 1 problem in 1 formula
homebrew/php/php70-uopz:
  * Description is too long. "name: desc" should be less than 80 characters.
    Length is calculated as php70-uopz + desc. (currently 90)
```

**Comment by SMillerDev** [comment]
@javian Fixed now, not sure how to mark that though

**Comment by javian** [comment]
@SMillerDev I'm on it thanks!

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by javian** [comment]
Thanks @SMillerDev ! Would you mind adding a formula for 7.1 as well ?

**Comment by SMillerDev** [comment]
couldn't get it working on 7.1 yet, I'll give it another try but not holding my breath.

**Pull request closed without merging**